.github/workflows/tests.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       mysql:
         image: mysql:5.7
         env:
-          MYSQL_ROOT_PASSWORD: root
+          MYSQL_ROOT_PASSWORD: root#
```

Comment:
test workflow also takes care of this. ENV uses the original password (with reserved chars)